src/main/java/org/kainos/ea/FloorIsJavaApplication.java

```diff
@@ -30,8 +36,9 @@ protected SwaggerBundleConfiguration getSwaggerBundleConfiguration(FloorIsJavaCo
 
     @Override
     public void run(final FloorIsJavaConfiguration configuration,
-                    final Environment environment) {
+                    final Environment environment) throws NoSuchAlgorithmException {
```

Comment:
don't throw this exception from here, it should be caught and returned as a 500 in the controller

Reply:
Left this as is after discussion